package.json

```diff
@@ -125,23 +125,23 @@
     "gulp-remote-src": "^0.4.4",
     "gulp-rename": "^1.2.0",
     "gulp-replace": "^0.5.4",
-    "gulp-shell": "^0.6.5",
+    "gulp-shell": "^0.7.0",
     "gulp-tsb": "2.0.7",
     "gulp-tslint": "^8.1.3",
     "gulp-uglify": "^3.0.0",
-    "gulp-untar": "^0.0.7",
+    "gulp-untar": "^0.0.8",
     "gulp-vinyl-zip": "^2.1.2",
     "husky": "^0.13.1",
     "innosetup-compiler": "^5.5.60",
     "is": "^3.1.0",
-    "istanbul": "^0.3.17",
+    "istanbul": "^0.4.5",
     "jsdom-no-contextify": "^3.1.0",
     "lazy.js": "^0.4.2",
     "merge-options": "^1.0.1",
     "mime": "^1.4.1",
     "minimatch": "^3.0.4",
     "mkdirp": "^0.5.0",
-    "mocha": "^2.2.5",
+    "mocha": "^6.1.4",
```

Comment:
You have to be careful when updating this far. It very likely could have breaking changes.